### PR TITLE
top level folder ui change fix

### PR DESCRIPTION
**Pull request by thanhz**

### CODE REVIEW

File: src/test/scala/helpers/steps/StepsUtility.scala [hunk 1]
```diff
@@ -52,9 +54,16 @@ object StepsUtility {
   }
 
   def elementIsHidden(id: String, webDriver: WebDriver): Boolean = {
-    val element = webDriver.findElement(By.cssSelector(s"#$id"))
-    val hideCssClasses = List("hide", "govuk-visually-hidden")
-    hideCssClasses.exists(element.getAttribute("class").contains) || Option(element.getAttribute("hidden")).isDefined
+    val findElementByIdOrClass = {
+      Try(webDriver.findElement(By.cssSelector(s"#$id")))
+        .orElse(Try(webDriver.findElement(By.cssSelector(s".$id"))))
```
Comment (vimleshtna) [inline src/test/scala/helpers/steps/StepsUtility.scala:59]: You can use the "," for multiple selectors.

`By.cssSelector(s"#$id,.$id")`

**Review by vimleshtna — COMMENTED** [review]
Looks good. Just a minor comment. Thanks

File: src/test/scala/helpers/steps/StepsUtility.scala (hunk 1 above)
Comment (thanhz) [reply src/test/scala/helpers/steps/StepsUtility.scala]: that's more simple thanks.

**Review by vimleshtna — APPROVED** [review]
Looks good. Thanks

**Pull request merged**